src/dictionary/domain/compareAkkadianStrings.ts

```diff
@@ -57,3 +57,9 @@ export default function compareAkkadianStrings(
       .head() ?? Math.sign(replacedWord.length - anotherWordReplaced.length)
   )
 }
+export function cleanAkkadianString(akkadianString: string): string {
+  return akkadianString
+    .split('')
+    .map((signChar) => (alphabet.indexOf(signChar) > 0 ? signChar : ''))
```

Comment:
Why is the first alphabet cleaned out?

Reply:
comparing them errors on non-akkadian letters and that 's why i make sure there are only akkadian letters (for comparing, sorting later on).